core/mgmt/v1beta/metric.proto

```diff
@@ -180,11 +180,12 @@ message ListCreditConsumptionChartRecordsRequest {
   // Aggregation window. The value is a positive duration string, i.e. a
   // sequence of decimal numbers, each with optional fraction and a unit
   // suffix, such as "300ms", "1.5h" or "2h45m".
-  string aggregation_window = 2 [(google.api.field_behavior) = REQUIRED];
-  // Filter can hold an [AIP-160](https://google.aip.dev/160)-compliant filter
-  // expression to determine the time range for the charts.
-  // - Example: `start='2024-06-04T10:000:00Z' AND stop='2024-06-11T10:00:00Z`
-  optional string filter = 3 [(google.api.field_behavior) = OPTIONAL];
+  // The minimum (and default) window is 1h.
```

Comment:
@donch1989 @EiffelFly one last change. The transpiler in `mgmt-backend` produced only an empty expression, even if I copied the filter from the existing dashboard endpoints. In order to make it and, given that all the designs we're handling right now only filter by start and end date, I removed the `filter` param